src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -158,53 +166,78 @@ static void uct_rdmacm_iface_process_conn_req(uct_rdmacm_iface_t *iface,
     rdma_destroy_id(event->id);
 }
 
-static void uct_rdmacm_iface_process_event(uct_rdmacm_iface_t *iface, struct rdma_cm_event *event)
+static void uct_rdmacm_iface_return_cm_id(uct_rdmacm_iface_t *iface,
```

Comment:
uct_rdmacm_iface_release_cm_id
pls add comment that is should be called when async context is locked

Reply:
you mean in case someone would want to call this function from the main thread?

Reply:
yes